api/aqua-api-example/index.js

```diff
@@ -0,0 +1,30 @@
+import {
+    Aqua,
+    Call,
+    Path,
+} from "@fluencelabs/aqua-api/aqua-api.js";
+import { FluencePeer } from "@fluencelabs/fluence";
+import { callFunctionImpl } from "@fluencelabs/fluence/dist/internal/compilerSupport/v3impl/callFunction.js";
```

Comment:
oh. Any way to avoid using `callFunctionImpl`?

I'm not sure that calling the function should be a part of Aqua **compiler** API at all.